jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ExtraDirectoryParameters.java

```diff
@@ -18,60 +18,47 @@
 
 import com.google.cloud.tools.jib.plugins.common.RawConfiguration.ExtraDirectoriesConfiguration;
 import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.util.List;
 import javax.inject.Inject;
-import org.gradle.api.Project;
+import org.gradle.api.file.ConfigurableFileCollection;
 import org.gradle.api.model.ObjectFactory;
 import org.gradle.api.provider.ListProperty;
 import org.gradle.api.provider.Property;
 import org.gradle.api.provider.Provider;
 import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.InputFiles;
 import org.gradle.api.tasks.Internal;
 
 /** Configuration of an extra directory. */
 public class ExtraDirectoryParameters implements ExtraDirectoriesConfiguration {
 
-  private Project project;
-  private Property<Path> from;
+  ConfigurableFileCollection from;
```

Comment:
Should this type should be `DirectoryProperty` instead of `ConfigurableFileCollection`?

Reply:
Unfortunately the implementation of DirectoryProperty is a bit of a mess (see: https://github.com/gradle/gradle/issues/23708). Might be able to convert it but it would be pretty messy

Reply:
Kk.